src/Stratis.Bitcoin.Features.SmartContracts.Wallet/ContractWalletHistoryModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Stratis.Bitcoin.Features.SmartContracts.Wallet
+{
+    public class ContractWalletHistoryModel
+    {
+        public ContractWalletHistoryModel()
+        {
+            this.AccountsHistoryModel = new List<ContractAccountHistoryModel>();
+        }
+
+        [JsonProperty(PropertyName = "history")]
+        public ICollection<ContractAccountHistoryModel> AccountsHistoryModel { get; set; }
```

Comment:
Each wallet history request specifies the account name so I assume this should only ever have one item? Feels like unnecessary nesting if so.